test_othermlpackages/test_tensorflow.py

```diff
@@ -1,15 +1,19 @@
 # Copyright (c) Microsoft Corporation and Fairlearn contributors.
 # Licensed under the MIT License.
 
+
 import pytest
 from . import package_test_common as ptc
+import scikeras
```

Comment:
Is this actually used?

Reply:
Yes, scikeras has been used to import KerasClassifier, which was previously being called with the help of `tf.keras.wrappers.scikit_learn.KerasClassifier`.

Reply:
OK. Perhaps you could move this adjacent to the `scikeras.wrappers` import below?

Reply:
@riedgar-ms Okay. I'll remove this and only keep the import of `scikeras.wrappers` in the updated PR.